stdlib/kokorawr/kokorawr/functions/services/rps.js

```diff
@@ -23,14 +25,10 @@ module.exports = (user='Bob', team=1, option='rock', context, callback) => {
     if (!gameInfo.hasOwnProperty('waitingTeam')) {
       gameInfo['waitingTeam'] = -1;
     }
-    if (!gameInfo.hasOwnProperty('options')) {
-      gameInfo['options'] = ['rock', 'paper', 'scissors'];
-    }
     let waiting = gameInfo['waiting'];
     let waitingTeam = gameInfo['waitingTeam'];
-    let options = gameInfo['options'];
 
-    let index = options.indexOf(option.toLowerCase());
+    let index = options.indexOf(option.toLowerCase()) % 3;
```

Comment:
You don't have to do this mod 3 because it should always be inside of the options. If it is not then it should give you a -1 in which case you can just throw an error

Reply:
Mod 3 was used so the indices will only ever be 0, 1, or 2. Thus making index of 'rock' equal to index of 'r' so itll be easier to do the win check later with only 3 indices instead of 6

Reply:
Oh I see now.... this is still very strange and I don't really like it lol. We could just use the first character of the option passed in then

Reply:
If the user passes in 'statue', it'll use the first letter which is 's' and will treat it as scissors.

Reply:
it should get stopped at the command function

Reply:
fine you can leave this

Reply:
ok 😄